pyproject.toml

```diff
@@ -1,61 +1,65 @@
 [build-system]
 build-backend = "hatchling.build"
-requires = ["hatchling>=1.12.2", "hatch-vcs>=0.3"]
+requires = [
+  "hatch-vcs>=0.3",
+  "hatchling>=1.18",
+]
 
 [project]
 name = "pytest-memray"
 description = "A simple plugin to use with pytest"
-readme.file = "README.md"
 readme.content-type = "text/markdown"
+readme.file = "README.md"
```

Comment:
Why was this line moved? Did a tool do this automatically?

Reply:
Indeed used https://github.com/tox-dev/pyproject-fmt